release/staging-build.yml

```
@@ -205,3 +205,9 @@ batch:
     depend-on:
     - cri_tools
     - etcdadm
+  - identifier: eks-anywhere-packages
```

Identifier names cannot have dashes.
```suggestion
  - identifier: eks_anywhere_packages
```

(I know right 🙄)